cmd/restic/global.go

```diff
@@ -584,10 +596,18 @@ func parseConfig(loc location.Location, opts options.Options) (interface{}, erro
 			cfg.AccountName = os.Getenv("AZURE_ACCOUNT_NAME")
 		}
 
+		if cfg.AccountName == "" {
```

Comment:
The azure storage clients itself already complains if the account name / key are not set (although it can't print the name of the environment variable)